src/main/java/com/amazon/opendistroforelasticsearch/security/securityconf/DynamicConfigFactory.java

```diff
@@ -320,5 +328,39 @@ public String getHash(String user) {
             return Collections.emptyList();
         }
     }
+
+    private static class NodesDnModelV7 extends NodesDnModel {
```

Comment:
What is the difference between `NodesDnModelV6` and `NodesDnModelV7`?

Reply:
> +1 about difference question,
> 
> and as this is new configuration, do we really need V6 as well? Are we supporting this for 6x and needed for 6.8 to 7.x migration?

Yes, will be supporting 6.7+

Reply:
Merged the classes.